client/layout/sidebar/menu.jsx

```diff
@@ -2,9 +2,10 @@
  * External dependencies
  */
 import React from 'react';
+import classNames from 'classnames';
 
-const SidebarMenu = ( { children } ) => (
-	<li className="sidebar__menu">{ children }</li>
+const SidebarMenu = ( { children, className } ) => (
```

Comment:
The class names on the component. What are we using it for? Do we need the class name?